### ISSUE TITLE
Benchmark

### ISSUE THREAD

**Issue by hafidh561** [unifyai/ivy#1295]
Hello, I think ivy is such a fantastic library deep learning for univy our code. But, there is something I want to ask, is about benchmarking with the native library. Is there any benchmark for ivy vs native pytorch maybe? or tensorflow, etc. Maybe benchmark for speed, accuracy, etc.

**Comment by jkeane508** [comment]
Hi,

Currently there are no benchmarks for the performance of Ivy using different backends, but we will add these benchmarks as soon as they are ready. However, during early experimentation, we observed runtime overhead in the order of ~1% when compared to the backends directly. 🚀

I hope this answers your question!
